src/en/hiveworks/src/eu/kanade/tachiyomi/extension/en/hiveworks/Hiveworks.kt

```diff
@@ -178,8 +178,27 @@ class Hiveworks : ParsedHttpSource() {
     // Included to call custom error codes
     override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
         return if (manga.status != SManga.LICENSED) {
+            val uri = Uri.parse(manga.url).buildUpon()
             client.newCall(chapterListRequest(manga))
-                .asObservableSuccess()
+                .asObservable()
+                .doOnNext { response ->
+                    if (!response.isSuccessful) {
+                        if ("smbc-comics" in uri.toString() && response.code == 500) {
+                            // As of March 2025, SMBC chapter list page returns
+                            // status code 500 even though it still has correct data.
+                            // Do not throw an error in this case.
+                            //
+                            // I reported this error to SMBC on 2025-05-28, but even if
+                            // it is fixed, the same problem might occur again in the future.
+                            return@doOnNext
+                        }
+                        // Otherwise, an unsuccessful response status indicates that
+                        // there is an error. Apparently, HttpException isn't able to
+                        // be imported from extensions-lib, so we have to copy and paste.
```

Comment:
I don't think this context (comments) is needed.

Reply:
If you meant to remove both comments, and not just the second one, let me know and I can do that.